video/out/vo_x11.c

```diff
@@ -102,16 +118,19 @@ static bool getMyXImage(struct priv *p, int foo)
             MP_WARN(vo, "Shared memory error,disabling ( Ximage error )\n");
             goto shmemerror;
         }
-        p->Shminfo[foo].shmid = shmget(IPC_PRIVATE,
-                                       p->myximage[foo]->bytes_per_line *
-                                       p->myximage[foo]->height,
-                                       IPC_CREAT | 0777);
+        len = p->myximage[foo]->bytes_per_line * p->myximage[foo]->height;
+        memcpy(p->shmname, "/tmp/mpv-x11-XXXXXXXXXX", sizeof(p->shmname));
+        p->Shminfo[foo].shmid = shm_mkstemp(p->shmname);
```

Comment:
another function that does not exist on Linux

Reply:
Yes, I was not aware of this being an extension function until after it went through CI.